packages/katello/rubygem-katello/rubygem-katello.spec

```
@@ -42,18 +42,18 @@ Requires: %{?scl_prefix}rubygem(apipie-rails) >= 0.5.14
 Requires: %{?scl_prefix}rubygem(runcible) >= 2.13.0
 Requires: %{?scl_prefix}rubygem(runcible) < 3.0.0
 Requires: %{?scl_prefix}rubygem(anemone)
-Requires: %{?scl_prefix}rubygem(pulpcore_client) >= 3.3.0
-Requires: %{?scl_prefix}rubygem(pulpcore_client) < 3.4.0
-Requires: %{?scl_prefix}rubygem(pulp_file_client) >= 0.3.0
-Requires: %{?scl_prefix}rubygem(pulp_file_client) < 0.4.0
+Requires: %{?scl_prefix}rubygem(pulpcore_client) >= 3.4.0
+Requires: %{?scl_prefix}rubygem(pulpcore_client) < 3.5.0
+Requires: %{?scl_prefix}rubygem(pulp_file_client) >= 1.0.0
+Requires: %{?scl_prefix}rubygem(pulp_file_client) < 1.1.0
 Requires: %{?scl_prefix}rubygem(pulp_ansible_client) >= 0.2.0
 Requires: %{?scl_prefix}rubygem(pulp_ansible_client) < 0.3.0
 Requires: %{?scl_prefix}rubygem(pulp_certguard_client) < 2.0
-Requires: %{?scl_prefix}rubygem(pulp_container_client) >= 1.3.0
-Requires: %{?scl_prefix}rubygem(pulp_container_client) < 1.4.0
-Requires: %{?scl_prefix}rubygem(pulp_rpm_client) >= 3.3.0
-Requires: %{?scl_prefix}rubygem(pulp_rpm_client) < 3.4.0
-Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) >= 0.2.0
+Requires: %{?scl_prefix}rubygem(pulp_container_client) >= 1.4.0
+Requires: %{?scl_prefix}rubygem(pulp_container_client) < 1.5.0
+Requires: %{?scl_prefix}rubygem(pulp_rpm_client) >= 3.4.0
+Requires: %{?scl_prefix}rubygem(pulp_rpm_client) < 3.5.0
+Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) >= 0.2.0b2
```

```suggestion
Requires: %{?scl_prefix}rubygem(pulp_2to3_migration_client) >= 0.2.0
```

(our RPM packages don't/can't follow GEM versioning)